web/templates/popovers/stream_card_popover.hbs

```diff
@@ -12,6 +12,13 @@
             {{> ../stream_settings/stream_description
               rendered_description=stream.rendered_description}}
         </li>
+        {{#if subscribers_count}}
```

Comment:
Hi @evykassirer ,

**Yes, it can indeed be falsy** if there are no subscribers in a stream. For instance, if a user creates a stream and is the only subscriber, and then unsubscribes, this line won’t render in the popover card. I’ve added screenshots below to illustrate both cases—one with a subscriber and another without.

| Popover with subscriber(s) | Popover without subscriber |
|--|--|
| ![stream-card-popover-with-subscriber](https://github.com/user-attachments/assets/230a8a7b-cb52-434b-95e1-f5415a32058d) | ![stream-card-popover-without-subscriber](https://github.com/user-attachments/assets/f28d2679-d3db-4d35-83a9-3cb2e31c6335) |

Thank you for reviewing!

Reply:
Ok great, looks good!

Unless we want to specifically say "No subscribers" or "0 subscribers" there?

Reply:
This would be good to ask about on CZO! You can start a thread in #design.

Reply:
@evykassirer, as per the recent discussion in this thread, I have updated the code to check for 0 subscribers and render 'No Subscribers.' Kindly review the changes.